model.js

```diff
@@ -96,3 +96,43 @@ exports.selectReviewComments = (reviewId) => {
       return result.rows;
     });
 };
+exports.writeReviewComments = (review_id, username, body) => {
```

Comment:
i suggest changing this function name from 'writeReviewComments' to something like 'insetReviewComments' just to avoid any confusion because this function doesn't write a comment, it adds one ⭐